agreement-view-component.html

```diff
@@ -0,0 +1,18 @@
+<stackable-modal stackable="$ctrl.visible.agreementModal">
+  <br-modal br-title="{{$ctrl.agreementHeader}}">
+    <br-modal-body>
+        <br-agreement
+          ng-model="$ctrl.agreed"
+          br-agreement-group="{{$ctrl.group}}"
+          br-show-header="false">
+        </br-agreement>
+        <br><br>
+    </br-modal-body>
+    <br-modal-footer>
+      <button ng-disabled="!$ctrl.agreed" type="button" class="btn btn-primary"
+        ng-click="$ctrl.done()">
+        Done
```

Comment:
Maybe "Confirm". We should also have text that says: "You must confirm that you accept the following agreements in order to continue." if we don't already.